dunstrc

```diff
@@ -258,6 +258,17 @@
     # Context menu.
     context = ctrl+shift+period
 
+# Defines action of mouse event
+# Possible values are:
+# * do_action: If the notification has exactly one action, or one is marked as default,
+#              invoke it. If there are multiple and no default, open the context menu.
+# * close_current: Close current notification.
+# * push_all: Push all waiting and displayed notifications to history.
+[mouse]
```

Comment:
I'd prefer it if this was in the global section as well as `mouse_left_click, mouse_middle_click` etc. I think sections are best reserved for rules/more customization down the line rather than setting groups.